definitions.yaml

```diff
@@ -523,6 +526,12 @@ definitions:
       owner:
         type: string
         description: Name of the organization owning the cluster
+      availability_zones:
```

Comment:
Unsure if this is necessary in the list of clusters. Kind of a UX decision but I don't see a list of possibly 3+ AZ names as useful here.

Reply:
Maybe to add a column with the number of AZs